content/ops/aws-onboarding.md

```diff
@@ -5,10 +5,38 @@ menu:
 title: AWS Onboarding
 ---
 
+The procedures for working with Amazon Web Services (AWS) differ between the two environments, and thus members of Atlas are given different levels of access.
+
+### US-East
+
 If you are going to be a Cloud Foundry operator you need to be onboarded to AWS.
 
 Things you will need:
 
 - Access to EC2, RDS and CloudFormation
 - Your key added to the jumpbox
 - The IP of the jumpbox (look at the CloudFormation stack for it)
+
+### GovCloud
+
+<!-- needed for compliance -->
+
+#### Levels of access
+
+While we try to minimize interaction with AWS directly (explained in next section), there are three different levels of AWS IAM access, provided to different roles:
+
+* **Full administrative accounts:** This is be provided to the people known in our compliance documentation as the "Cloud Operations team".
+* **Read-only accounts:** This is provided to GSA Security staff, who need access for incident response.
+* **Indirect access through jumpboxes:** While the remainder of the Atlas team isn't given an AWS IAM account directly, they are able to SSH in to access the various BOSH machines by creating jumpboxes. See the [Troubleshooting BOSH](../troubleshooting-bosh/) page for more information.
```

Comment:
s/jumpboxes/ephemeral jumpboxes/ (just to make the point that they're not setting up their own personal jumpboxes)